brayns/utils/image/ImageInfo.cpp

```diff
@@ -45,6 +47,11 @@ size_t ImageInfo::getPixelSize() const
 
 size_t ImageInfo::getPixelIndex(size_t x, size_t y) const
 {
+    if (x >= width || y >= height)
+    {
+        throw std::out_of_range("Image coordinates out of bounds");
+    }
```

Comment:
Or assert (like you prefer but I was thinking about it like for std::vector, we assume the user provides correct inputs to avoid runtime overhead in release mode).

Reply:
Same reply I made above.